lib/onelogin/ruby-saml/response.rb

```diff
@@ -27,19 +38,32 @@ def initialize(response, options = {})
         @document = XMLSecurity::SignedDocument.new(@response, @errors)
       end
 
+      # An aux function to validate the SAML Response with the default values (soft = true)
```

Comment:
no reason to have these comments `An aux function...`.  Keep documentation short and sweet